sonar-plugin-api/src/main/java/org/sonar/api/batch/sensor/highlighting/internal/DefaultHighlighting.java

```diff
@@ -79,6 +79,8 @@ public DefaultHighlighting highlight(int startOffset, int endOffset, TypeOfText
     checkInputFileNotNull();
     TextRange newRange;
     try {
+      // TODO remove when SONAR-7664 is implemented
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1135)